src/components/KSlideout/KSlideout.vue

```diff
@@ -1,79 +1,59 @@
 <template>
   <div class="k-slideout">
-    <transition name="fade">
+    <Transition name="kongponents-fade-transition">
       <div
-        v-if="isVisible"
-        :class="hasOverlay ? 'panel-background' : 'panel-background-transparent'"
+        v-if="visible"
+        class="slideout-backdrop"
+        :class="{ 'backdrop-transparent': !hasOverlay }"
       />
-    </transition>
-    <transition name="slide">
+    </Transition>
+    <Transition name="kongponents-slide-in-transition">
       <div
-        v-if="isVisible"
-        ref="slideOutRef"
-        class="panel"
-        :class="{ 'is-visible': isVisible, 'border-styles': !hasOverlay }"
-        data-testid="slideout-panel"
+        v-if="visible"
+        ref="slideoutContainerElement"
+        class="slideout-container"
+        data-testid="slideout-container"
       >
-        <div class="k-slideout-header-content">
+        <div class="slideout-header">
           <div
-            v-if="hasBeforeTitle"
-            class="k-slideout-before-title"
+            v-if="title || $slots.title"
+            class="slideout-title"
+            data-testid="slideout-title"
           >
-            <slot name="before-title" />
-          </div>
-
-          <!-- title -->
-          <div class="k-slideout-main-title">
-            <p
-              class="k-slideout-title"
-              data-testid="k-slideout-title"
-              :title="title"
-            >
+            <slot name="title">
```

Comment:
@johncowen @kleinfreund based on conversation in https://github.com/kumahq/kuma-gui/pull/2184, if I understand it correctly, these's two outstanding issues with the reskinned KSlideout:
1. Title prop and slot content styles are enforced which makes it harder to make the title custom (e.g. link in your example)
2. The `<body>` element becomes non-scrollable one KSlideout is open, which makes scrolling content in the background impossible

Did I miss anything? Just want to lay questions out before addressing them

Reply:
Removing this feature from KSlideout. At first I and Adam thought it's a good idea to make body non-scrollable but we forgot to account for the fact that KSlideout is often used where we still want the content underneath it to be viewable.

Reply:
As for my first point, there's nothing we want to do from KSlideout side - it will ship with pre-defined styles for title for consistency and if you need to use your custom title - you might have to overwrite those styles here and there.

Reply:
Gotcha, sounds good to me 👍 let us know if we can help/test anything new in regards to the scroll behaviour

Reply:
Approved that one, watching here and will try it out when its on here.

Reply:
Thank you! Merged now

Reply:
Updated https://github.com/kumahq/kuma-gui/pull/2184 and this is working now.

Reply:
Looks much better to me. I do still get the double scrollbar if I manage to make the KSlideout content short enough to make it scroll. But I have to try really hard to do that, especially considering the content we put in these at the moment is relatively small. Plus, really you want to be able to scroll both things separately so its kinda hard to do anything better in this respect I think.

So all in all, I definitely think we've optimised for the most best and common case here 🙌 

@kleinfreund I tried this on my little scratch branch I had, does this look good to you also?